jolt-core/src/main/java/com/bazaarvoice/jolt/templatr/TemplatrSpecBuilder.java

```diff
@@ -20,15 +20,17 @@
 import com.bazaarvoice.jolt.templatr.spec.TemplatrCompositeSpec;
 import com.bazaarvoice.jolt.templatr.spec.TemplatrContextLookupLeafSpec;
 import com.bazaarvoice.jolt.templatr.spec.TemplatrDefaultLeafSpec;
+import com.bazaarvoice.jolt.templatr.spec.TemplatrFunctionLeafSpec;
 import com.bazaarvoice.jolt.templatr.spec.TemplatrSelfLookupLeafSpec;
 import com.bazaarvoice.jolt.templatr.spec.TemplatrSpec;
 
 import java.util.Map;
 
 public class TemplatrSpecBuilder extends SpecBuilder<TemplatrSpec> {
 
-    private static final String CARET = "^";
-    private static final String AT = "@";
+    public static final String CARET = "^";
+    public static final String AT = "@";
+    public static final String FUNCTION = "function:";
```

Comment:
So was thinking maybe "_" or "." as the signal of a function.

"maxScore" : ".max(@(1,scores))"  or 
"maxScore" : "_max(@(1,scores))"

I kinda like the starting "."